docs/reference/content/driver/tutorials/ssl.md

```diff
@@ -201,3 +201,49 @@ ERefGuide.html).
 Some applications may want to force only the TLS 1.2 protocol. To do this, set the `jdk.tls.client.protocols` system property to "TLSv1.2".
 
 Java runtime environments prior to Java 8 started to enable the TLS 1.2 protocol only in later updates, as shown in the previous section. For the driver to force the use of the TLS 1.2 protocol with a Java runtime environment prior to Java 8, ensure that the update has TLS 1.2 enabled.
+
+
+## OCSP
+
+{{% note %}}
+The Java driver cannot enable OCSP by default on a per MongoClient basis.
+{{% /note %}}
+
+### Client-driven OCSP
+
+An application will need to set JVM system and security properties to ensure that client-driven OCSP is enabled:
+
+-  `com.sun.net.ssl.checkRevocation`:
+      When set to `true`, this system property enables revocation checking.
+
+-  `ocsp.enable`:
+      When set to `true`, this security property enables client-driven OCSP.
+      
+To configure an application to use client-driven OCSP, the application must already be set up to connect to a server using TLS. Setting these system properties is required to enable client-driven OCSP.
+
+### OCSP Stapling
+
+{{% note class="important" %}}
+Red Hat Enterprise Linux 7 does not support OCSP stapling.
```

Comment:
This may require some additional clarification:
I also believe that an application using the Java driver running on RHEL7 should be able utilize stapled OCSP: it's the `mongod` that doesn't support stapled OCSP on RHEL7 (due to the openSSL version that ships with RHEL7, as I understand it.

Nit: OCSP stapling => stapled OCSP?